src/replay/commit.zig

```diff
@@ -89,6 +102,38 @@ pub const Committer = struct {
             );
             try self.account_store.put(slot, pubkey, account);
         }
+
+        for (transactions, tx_results) |transaction, result| {
+            _, const tx_result = result;
+
+            switch (tx_result) {
+                .executed => |exec| {
```

Comment:
the current function already has this code. see line 79. we don't need another loop and switch